src/distdl/nn/conv_general.py

```diff
@@ -76,6 +77,13 @@ def __init__(self, P_x, P_y, P_w,
         # P_w is P_co x P_ci x P_d-1 x ... x P_0
         self.P_w = P_w
 
+        # Back-end specific buffer manager for economic buffer allocation
+        if buffer_manager is None:
+            buffer_manager = self._distdl_backend.BufferManager()
+        elif type(buffer_manager) is not self._distdl_backend.BufferManager:
+            raise ValueError("Incompatible buffer manager.")
```

Comment:
Again nitpicky, but this error could be a bit more specific, since the condition is checking is "Is the buffer manager the same as the one in the backend"

Reply:
I thought about this when implementing it and decided against it.  The more I think on it the more I agree, though, so I'll make that change shortly.

Reply:
Done.